src/Plugin/feeds_migrate/data_fetcher/File.php

```diff
@@ -83,7 +83,7 @@ public function alterMigration(FeedsMigrateImporterInterface $importer, Migratio
       if ($file = FileEntity::load(reset($fids['file']))) {
 
         $source_config = $migration->getSourceConfiguration();
-        $source_config['urls'] = file_create_url($file->getFileUri());
+        $source_config['urls'] = \Drupal::service('file_system')->realpath($file->getFileUri());
```

Comment:
Maybe add a todo to use DI for this service?

Reply:
Added DI.